OpenRA.Mods.Common/Scripting/Global/MediaGlobal.cs

```diff
@@ -86,12 +86,28 @@ public void PlayMusic(string track = null, LuaFunction func = null)
 				playlist.Play(musicInfo);
 		}
 
+		[Desc("Play track defined in music.yaml as background music.")]
+		public void PlayBackgroundMusic(string track)
+		{
+			if (!playlist.IsMusicAvailable)
+				return;
+
+			playlist.Play(world.Map.Rules.Music[track]);
```

Comment:
I tested this and it works.
As `Play(MusicInfo music)` sets `CurrentSongIsBackground` to `false`, I set it to `true` below.
`Play(MusicInfo music)` calls `Play()` and this uses
```
Game.Sound.PlayMusicThen(currentSong, () =>
{
	if (!CurrentSongIsBackground && !Game.Settings.Sound.Repeat)
		currentSong = GetNextSong();

	Play();
});
```
By then `CurrentSongIsBackground` is `true` (because of my line below) and the song is repeated.